src/common/services/api/designations.service.js

```diff
@@ -158,13 +160,13 @@ class DesignationsService {
       })
 
       return {
-        uri: this.hateoasHelperService.getLink(data.definition, 'item'),
+        uri: this.hateoasHelperService.getLink(data.rawData, 'addtocartform'),
```

Comment:
Not sure why we're using the `addtocartform` uri on product lookup. This service function is for displaying the properties of the designation to the user.

Reply:
@sateeshbattuta or @tadkbabu

Reply:
For this, we have changed the edit cart functionally based on the EP8.1 JSON object. Now it is working as expected. Please check.